### DISCUSSION TITLE
Social share card for root of website?

### DISCUSSION THREAD

**Discussion by bee-san** [adityatelange/hugo-PaperMod#157] [Q&A] [answered]
When sharing the blog on Twitter, the card looks like this:
![image](https://user-images.githubusercontent.com/10378052/103170562-03452480-483d-11eb-8b95-cf686dcc01a8.png)

It's possible to make it a large summary card by adding an image, see:
![image](https://user-images.githubusercontent.com/10378052/103170582-1fe15c80-483d-11eb-9a22-37493fdb8d2a.png)

How would I add a social share image  to the root of the hugo-papermod theme?

**Comment by adityatelange** [comment, marked as answer]
https://github.com/rizinorg/website/blob/6facbbdcf983536c46b79b36e6a4055a7999aa64/config.yml#L32-L33

this way
